CHANGELOG.md

```diff
@@ -1,3 +1,84 @@
+## v0.26.0 (2024-03-07)
+
+> **Note:**
+> - JSON-RPC API has been removed completely. Instead we recommend you to utilize REST API endpoints that have same and extended functionality.
+> Please have a look at Waku's REST-API reference: https://waku-org.github.io/waku-rest-api
+> - Support for Cross-Origin-Resource-Sharing (CORS headers) is added for our REST-API services. This allows you to access our REST-API from a different domain from a browser.
```

Comment:
nitpick, reworking a little bit. Maybe that suggestion is not correct but the original one sounds a bit unclear.
```suggestion
> - Support for Cross-Origin-Resource-Sharing (CORS headers) is added for our REST-API services. This allows you to access our REST-API from a browser.
```

Reply:
Accept!